src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/__tests__/HealthCertificateValidationServiceValidationTests.swift

```diff
@@ -299,775 +169,50 @@ class HealthCertificateValidationServiceValidationTests: XCTestCase {
 					responseReport = report
 					expectation.fulfill()
 				case let .failure(error):
-					XCTFail("Test should not fail with error: \(error)")
-				}
-			}
-		)
-		
-		// THEN
-		waitForExpectations(timeout: .short)
-		guard let report = responseReport else {
-			XCTFail("report must not be nil")
-			return
-		}
-		XCTAssertEqual(report, .validationOpen(validationResults))
-	}
-	
-	// MARK: - Success (Fail)
-	
-	func testGIVEN_ValidationService_WHEN_SomeRuleIsFailed_THEN_FailedShouldBeReturned() throws {
```

Comment:
Is this test moved somewhere else?

Reply:
The refactoring was so big, the test moved 1000 lines to the bottom. So he is still there:
https://github.com/corona-warn-app/cwa-app-ios/blob/d3feafff70f31847196b61763b3fa8a22ea0d40e/src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/__tests__/HealthCertificateValidationServiceValidationTests.swift#L188